rendermodules/module/render_module.py

```diff
@@ -109,6 +109,19 @@ def output_tilespecs_to_stack(self, tilespecs, output_stack=None,
             output_stack, tilespecs, sharedTransforms=sharedTransforms,
             pool_size=pool_size, close_stack=close_stack, render=render)
 
+    def validate_tilespecs(self, input_stack, output_stack, z, render=None):
+        missing_ts = []
+
+        render = self.render if render is None else render
+
+        input_rs = renderapi.resolvedtiles.get_resolved_tiles_from_z(input_stack, z, render=render)
+        output_rs = renderapi.resolvedtiles.get_resolved_tiles_from_z(output_stack, z, render=render)
+        if len(input_rs.tilespecs) != len(output_rs.tilespecs): # pragma: no cover
```

Comment:
maybe instead of checking just len() check tileids in a set overlap way

Reply:
this is what I meant here:
```
in_ids = set([t.tileId for t in input_rs.tilespecs])                                                                               
out_ids = set([t.tileId for t in output_rs.tilespecs])                                                                             
if in_ids != out_ids:
    etc.
```

Reply:
ok... have added the changes